GetDKAN/data-catalog-components
# Dynamic component

## QA
- Create new dataset and don't rebuild public
- View search, you should see the dataset, and clicking the dataset should load the content
